src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+/* stylelint-disable font-family-name-quotes */
+/* stylelint-disable rule-empty-line-before */
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+a {
+  text-decoration: none;
+  color: inherit;
+}
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+.logo-link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+.menu-list {
+  display: flex;
+}
+.list-item {
+  color: black;
```

Comment:
The color for `.list-item` is set to black. Ensure this provides enough contrast against the background for accessibility.